tests/rptest/tests/mirror_maker_test.py

```diff
@@ -206,6 +211,7 @@ def target_group_equal():
             return target_group.partitions == source_group.partitions and target_group.name == source_group.name
 
         # wait for consumer group sync
-        wait_until(target_group_equal, 60, backoff_sec=5)
+        timeout = 600 if self.redpanda.dedicated_nodes else 60
```

Comment:
Is the logic here that the timeout differs by a factor of 10 because the messages _per partition_ differs by a factor of 10, even though the overall amount of data is bigger by a factor of 10000?

Reply:
No, the timeout was based on how long the test took on CDT. So its specific to the instance type used.

I typically do timeouts in 10s, 30s, 120s, 300s, 600s series but the actual test takes about 360s on the current default CDT settings. I can lower the timeout in the next push.

Reply:
ok -- if it's just set based on empirical experience that's okay, no need to mess with it.  We'll run on faster instances sometimes but I don't expect to run on slower instances.